### PR TITLE
Update @algo-builder/web package

### PR DESCRIPTION
**Pull request by ratik21**
## Proposed Changes

+ remove nullish coalaesing operator
+ add `forceCreate()`

### CODE REVIEW

File: packages/web/src/lib/wallectconnect-mode.ts [hunk 1]
```diff
@@ -39,13 +39,22 @@ export class WallectConnectSession {
...
+    return await this.create();
+  }
+
   /**
    * Create new session
    */
   async create (): Promise<void> {
     if (this.connector.connected) {
       console.warn(`A session is already active`);
-      return;
```
Comment (vuvth) [inline packages/web/src/lib/wallectconnect-mode.ts:48]: Why you delete this line?

Comment (robert-zaremba) [reply packages/web/src/lib/wallectconnect-mode.ts:48]: good question, probably we should  return a current session if possible.

Comment (ratik21) [reply packages/web/src/lib/wallectconnect-mode.ts:48]: right, this shouldn't be removed. thanks

File: packages/web/src/lib/wallectconnect-mode.ts [hunk 2]
```diff
@@ -39,6 +39,16 @@ export class WallectConnectSession {
     this.wcAccounts = this.connector.accounts;
   }
 
+  /**
+   * Kills a session if already exists, create new session
+   */
+  async forceCreate (): Promise<void> {
```
Comment (robert-zaremba) [inline packages/web/src/lib/wallectconnect-mode.ts:45]: how about adding a `force` parameter to the function below? By default it will be `false`

Comment (ratik21) [reply packages/web/src/lib/wallectconnect-mode.ts:45]: yes

Comment (robert-zaremba) [reply packages/web/src/lib/wallectconnect-mode.ts:45]: the idea was to remove this function.

**Review by robert-zaremba — APPROVED** [review]
pre-approving, let's remove `forceCreate` and use the argument in `create`

File: packages/web/src/lib/wallectconnect-mode.ts (hunk 2 above)
Comment (ratik21) [reply packages/web/src/lib/wallectconnect-mode.ts:45]: ah yes, i forgot to remove it.

**Pull request merged**